src/inausoft.netCLI/Deserialization/RegexOptionsDeserializer.cs

```diff
@@ -69,7 +77,7 @@ public object Deserialize(Type type, string optionsExpression)
                 }
                 else
                 {
-                    property.SetMethod.Invoke(command, new object[] { Convert.ChangeType(option.Groups[2].Value, property.PropertyType) });
+                    property.SetMethod.Invoke(command, new object[] { Convert.ChangeType(option.Groups[2].Value.Replace("\"", ""), property.PropertyType) });
```

Comment:
Not sure about this line. Will it remove all quotes from a string argument? May this cause unwanted side effects if my argument actually contains a quote somewhere in the middle etc?

Reply:
So as I said this Deserializer probably needs to be rewritten entirely. But for now this little adjustments makes it working. You can run test to validate this. I will adjust test so that it would be easier to test new deserializer - that will not be based on regexes.